rebar.config.script

```diff
@@ -121,13 +121,13 @@ DepDescs = [
 {folsom,           "folsom",           {tag, "CouchDB-0.8.3"}},
 {hyper,            "hyper",            {tag, "CouchDB-2.2.0-4"}},
 {ibrowse,          "ibrowse",          {tag, "CouchDB-4.0.1-1"}},
-{jaeger_passage,   "jaeger-passage",   {tag, "CouchDB-0.1.13-1"}},
+{jaeger_passage,   "jaeger-passage",   "7f56a93393070fb6e3778615a2d040573b10a6f3"},
```

Comment:
This sha would be updated to `{tag, "CouchDB-0.1.14-1"}` when apache/couchdb-jaeger-passage#1 is merged and correspondent tag created.

Reply:
Please merge that first, don't put shas into `master`.